Doc/library/asyncio-stream.rst

```diff
@@ -18,17 +18,16 @@ streams::
     import asyncio
 
     async def tcp_echo_client(message):
-        reader, writer = await asyncio.open_connection(
-            '127.0.0.1', 8888)
+        stream = await asyncio.connect('127.0.0.1', 8888)
```

Comment:
`connect` supports context manager:
```
async with asyncio.connect('127.0.0.1', 8888) as stream:
    await stream.write(...)
```
It can be more idiomatic that current version.
What do you think?

Reply:
+1 for context managers. I rewrote the examples and they are much more idiomatic and concise now.